### ISSUE TITLE
argopy notebook ok?

### ISSUE THREAD

**Issue by gaelforget** [euroargodev/ArgoData.jl#67]
The [argopy notebook](https://euroargodev.github.io/ArgoData.jl/dev/Argo_argopy.html) was recently updated to the new interface that is based on `PythonCall.jl` and `CondaPkg.jl` (in `ArgoData.jl v0.1.23` ).

There are a few ways the [Pluto](https://plutojl.org) notebook that seem like they could be improved without too much work. For example, a couple other functionalities of `argopy` could be demonstrated (code cells) and a bit more information on `argopy` added (markdown cells). 

@gmaze , are you able to run [the notebook](https://raw.githubusercontent.com/euroargodev/ArgoData.jl/refs/heads/master/examples/Argo_argopy.jl)?

**Comment by gmaze** [comment]
Yes I was able to run the notebook, although it took quite a long time to compile everything
You could add a link toward the argopy documentation: https://argopy.readthedocs.io
One of the most important feature of argopy is that it let users choose how much of the Argo jargon to see with the user mode

**Comment by gaelforget** [comment]
> Yes I was able to run the notebook, although it took quite a long time to compile everything

Great to hear! 

Should be faster the 2nd time around. The time to `compile` (or rather `compile+boot up`) here includes installing `argopy` and its dependencies , and downloading data to plot. Second time around it should skip installing `argopy` due to :

```
argopy=try
    ArgoData.pyimport(:argopy)
catch
    ArgoData.conda(:argopy)
    ArgoData.pyimport(:argopy)
end
```

> You could add a link toward the argopy documentation: https://argopy.readthedocs.io/

Technically there is one. But good point that more is needed.

> One of the most important feature of argopy is that it let users choose how much of the Argo jargon to see with the user mode

I need to explore more. Don't hesitate to send a PR on this
